apps/builder/app/builder/features/style-panel/sections/backgrounds/background-layers.test.ts

```diff
@@ -0,0 +1,177 @@
+import {
+  layeredBackgroundProps,
+  setLayerProperty,
+  addLayer,
+} from "./background-layers";
+
+import { describe, test } from "@jest/globals";
+import type { StyleInfo } from "../../shared/style-info";
+import type { CreateBatchUpdate } from "../../shared/use-style-data";
+import type {
+  LayersValue,
+  StyleProperty,
+  StyleValue,
+} from "@webstudio-is/css-data";
+
+// @todo remove at node18
```

Comment:
can we add it from some central place?

Reply:
why do you even need a structured clone for tests?

Reply:
I wasn't able to add it to jest via globals as usual. We need it for the test because it's used in the client code I'm testing.
In any case, we need to switch on the modern node and this will gone.

Reply:
needs an issue

Reply:
done https://github.com/webstudio-is/webstudio-builder/issues/1191